src/Bridge/Doctrine/Orm/Metadata/Property/DoctrineOrmPropertyMetadataFactory.php

```diff
@@ -15,7 +15,7 @@
 
 use ApiPlatform\Core\Metadata\Property\Factory\PropertyMetadataFactoryInterface;
 use ApiPlatform\Core\Metadata\Property\PropertyMetadata;
-use Doctrine\Common\Persistence\ManagerRegistry;
+use Symfony\Bridge\Doctrine\ManagerRegistry;
```

Comment:
I think you actually want `Doctrine\Persistence\ManagerRegistry` - that's the lower-level interface that we're *actually* requiring. See https://github.com/sensiolabs/SensioFrameworkExtraBundle/pull/691/files#diff-c7d9812ffa6b2809dabcf5120f2d2ec1 as an example

Reply:
Yes indeed that is a very good point. Using an abstract is way more restrictive than the interface itself. I though the bridge would manage the BC break by using the older or new version but it was an incorrect assumption.
I fear that Doctrine rollback the BC break which will again create some issue. I tried to use the method class_exists before using the use statement but that illegal in PHP. I'm a bit torn of the approach to take to be honest...

Reply:
It seems the issue also impact Doctrine\Persistence\ObjectManager; (previously `Doctrine\Common\Persistence\ObjectManager`).
I need to figure out how to run the tests on my machine before going further (missing mondo db on my machine is the issue I think)
```
ben@SB3:/tmp/test/test (master) $ vendor/bin/simple-phpunit --stop-on-failure -vvv vendor/api-platform/core/tests/
PHP Fatal error:  Trait 'ApiPlatform\Core\Tests\Bridge\Doctrine\Common\Filter\BooleanFilterTestTrait' not found in /tmp/test/test/vendor/api-platform/core/tests/Bridge/Doctrine/MongoDbOdm/Filter/BooleanFilterTest.php on line 25
```

Reply:
If you clone the project and if you launch `php -d memory_limit=-1 vendor/bin/phpunit tests/Bridge/Doctrine/MongoDbOdm/Filter` without the mongodb extension, there should be no problem.